addons/mail/static/src/core/common/composer.xml

```diff
@@ -38,7 +38,10 @@
                     }"
                 >
                     <div class="position-relative flex-grow-1">
-                        <textarea class="o-mail-Composer-input form-control bg-view px-3 border-0 rounded-3 shadow-none overflow-auto"
+                        <!-- classes relates to the spacing of textarea, so that the real and the fake are sharing the same spacing -->
```

Comment:
Comment is useless: there's already a comment on fake input to say it's used to resize input. The sharing part is obvious by seeting that both textarea share the classnames.

So comment just pollutes with an extra LOC, I'd remove it